### PR TITLE
Fixes #10398: Missing documentation for "default-directive-names.conf" 

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/10398

### CODE REVIEW

File: 4_advanced_usage/60_new_directives_default_name.txt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+That file need to be commited in git, and the Technique library reloaded 
+to take effect:
+
+---
+cd /var/rudder/configuration-repository/techniques/
+vi default-directive-names.conf
+ ....
+git add default-directive-names.conf
+git commit -m "Change default names for new directives"
+curl http://localhost:8080/rudder/api/techniqueLibrary/reload
```
Comment (amousset) [inline 4_advanced_usage/60_new_directives_default_name.txt]: rudder server reload-techniques

**Comment by fanf** [comment]
PR rebased

File: 4_advanced_usage/60_new_directives_default_name.txt [hunk 2]
```diff
@@ -0,0 +1,55 @@
+=== New directives default naming scheme
+
+When a new directive is created, by default the 'Name' field is filled
+with the Technique name. For example, if you create a new Directive from 
+the 'Users' Technique, the Name field will get the value: "Users"
```
Comment (amousset) [inline 4_advanced_usage/60_new_directives_default_name.txt]: . at the end

**Comment by fanf** [comment]
PR rebased

**Pull request merged**